### PR TITLE
zh-translation: /docs/examples/bookinfo/index.md

### PR DESCRIPTION
**Pull request by youhonglian**
- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/examples/bookinfo/index.md [hunk 1]
```diff
@@ -61,6 +68,10 @@ Bookinfo 应用中的几个微服务是由不同的语言编写的。
     $ kubectl label namespace default istio-injection=enabled
     {{< /text >}}
 
+    {{< warning >}}
+    如果您使用 OpenShift，请确保按照[OpenShift设置页面](/zh/docs/setup/platform-setup/openshift/#privileged-security-context-constraints-for-application-sidecars)中所述为名称空间上的服务帐户授予适当的权限
```
Comment (kebe7jun) [inline content/zh/docs/examples/bookinfo/index.md:72]:
```suggestion
    如果您使用 OpenShift，请确保按照[OpenShift设置页面](/zh/docs/setup/platform-setup/openshift/#privileged-security-context-constraints-for-application-sidecars)中所述为命名空间上的服务帐户授予适当的权限
```

File: content/zh/docs/examples/bookinfo/index.md [hunk 2]
```diff
@@ -157,21 +168,13 @@ $ curl -s http://${GATEWAY_URL}/productpage | grep -o "<title>.*</title>"
...
-* 如果**启用了**双向 TLS，请执行以下命令：
+{{< text bash >}}
+$ kubectl apply -f @samples/bookinfo/networking/destination-rule-all.yaml@
+{{< /text >}}
 
-    {{< text bash >}}
-    $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all-mtls.yaml@
-    {{< /text >}}
+{{< tip >}}
+`默认` 和 `示例` 的[配置文件](/zh/docs/setup/additional-setup/config-profiles/)默认情况下启用了[自动双向TLS](/zh/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls)。要执行双向 TLS，请使用 `samples/bookinfo/networking/destination-rule-all-mtls.yaml` 中的目标规则。
```
Comment (kebe7jun) [inline content/zh/docs/examples/bookinfo/index.md:176]:
```suggestion
`default` 和 `demo` 的[配置文件](/zh/docs/setup/additional-setup/config-profiles/)默认情况下启用了[自动双向TLS](/zh/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls)。要执行双向 TLS，请使用 `samples/bookinfo/networking/destination-rule-all-mtls.yaml` 中的目标规则。
```

**Review by kebe7jun — COMMENTED** [review]
/ok-to-test

**Comment by youhonglian** [comment]
@googlebot I signed it!

**Review by kebe7jun — APPROVED** [review]
LGTM

**Pull request merged**